elastic/eui
# Improve release script

## Summary

A couple optional extra things I noticed when working on #6471:

- Prettier lints the `release.js` file (vscode was doing this on every save for me otherwise)
- Fixes `path` undefined error when running `node scripts/release.js` directly (as opposed to `yarn release` or `npm run release`)
- Fixes a mysterious circular dependency warning (see below screenshot) by removing our `prompt` dependency (which was using the `winston` dependency causing the warning) in favor of [`inquirer`](https://www.npmjs.com/package/inquirer), which is more up to date & has nicer CLI APIs in any case (the major/minor/patch prompt now autofills! ✨)

I recommend following along by commit if necessary.

### Before

<img width="901" alt="" src="https://user-images.githubusercontent.com/549407/206582977-23fb8d4a-1c89-4818-83d7-883f6ba5b403.png">

<img width="898" alt="" src="https://user-images.githubusercontent.com/549407/206582862-3f535a6e-a39b-4e6b-bb35-062ec9fc9e94.png">

### After

![screencap](https://user-images.githubusercontent.com/549407/206582637-7187130e-a461-4b11-ae10-cd0c56649391.gif)

## QA

This is isn't super possible to QA locally without commenting out a whole bunch of stuff (the main branch check, the test/build steps for speed, definitely the npm publish step, etc) so I recommend doing code review alone on this and we'll QA test it on the fly during next week's release :shipit: 

### General checklist

N/A, dev-only
